models/model_upload/llms/sglang-llama-3_2-1b-instruct/requirements.txt

```diff
@@ -0,0 +1,18 @@
+torch==2.4
+tokenizers
+transformers==4.46.2
+accelerate
+optimum
+xformers
+einops
+requests==2.32.2
+packaging
+ninja
+protobuf==3.20.0
+
+sglang[all]
+orjson
+python-multipart
+
+--extra-index-url https://flashinfer.ai/whl/cu121/torch2.4/
```

Comment:
In prod and deb we have cuda 12.4, I'm not sure if it works with this `cu121`, need to be verified

Reply:
But I tested on q22 where prediction is successful, which also have cuda 12.4 but don;t thing this will be a issue